__tests__/app.test.js

```diff
@@ -32,3 +33,89 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("200: responds with an object describing all the available endpoints", () => {
```

Comment:
With the way that you are testing your endpoints file atm, you are going to have to update this every time you do a new endpoint. You can just require in the endpoints.json file into your tests and expect it to look like itself. That way every time you update it, this will be reflected in your tests and it will be dynamic

Reply:
Hi Saleh, I have revised the code for task 3 testing all the endpoint file on a different branch based on the feedback from another pull request. Thanks!